go-tipb/executor.pb.go

```diff
@@ -6643,6 +6685,57 @@ func (m *TopN) Unmarshal(dAtA []byte) error {
 				}
 			}
 		case 3:
+			if wireType != 2 {
```

Comment:
I would suggest another reviewer for this part.

Reply:
Same as above.